sentry_sdk/client.py

```diff
@@ -709,6 +721,7 @@ def capture_event(
 
         is_transaction = event_opt.get("type") == "transaction"
         is_checkin = event_opt.get("type") == "check_in"
+        is_error = not is_transaction and not is_checkin
```

Comment:
I guess this would also include profiles? Maybe make a check similar to the one for transaction?